tasks/aws/kind.py

```diff
@@ -59,6 +61,7 @@ def create_kind(
         key_pair_required=True,
         stack_name=stack_name,
         install_agent=install_agent,
+        install_agent_with_operator=install_agent_with_operator,
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
As this param is kind specific, I would keep it out of the `deploy` function, passing it through `extra_flags` instead
```go
    extra_flags = {
        "ddinfra:osDescriptor": f"amazonlinuxecs::{_get_architecture(architecture)}",
        "ddinfra:deployFakeintakeWithLoadBalancer": use_loadBalancer,
        "ddinfra:aws/defaultInstanceType": "t3.xlarge",
	"ddagent:deployWithOperator": install_agent_with_operator,
    }
```